.github/workflows/cupertino.yml

```diff
@@ -0,0 +1,56 @@
+name: package:cupertinohttp CI
+
+on:
+  # Run on PRs and pushes to the default branch.
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  schedule:
+    - cron: "0 0 * * 0"
+
+env:
+  PUB_ENVIRONMENT: bot.github
+
+jobs:
+  analyze:
+    name: Lint and static analysis
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: subosito/flutter-action@v2
+        with:
+          channel: 'stable'
+      - id: install
+        name: Install dependencies
+        run: flutter pub get
+        working-directory: pkgs/cupertinohttp
```

Comment:
As a drive by comment, you can move the cwd default up to the 'analyze' sections (as a sibling to the 'runs-on' config, and that will let you remove the explicit config from all the individual steps. There's an example here: https://github.com/google/flutter.widgets/blob/master/.github/workflows/build.yaml#L19

Reply:
Done.